js/app.js

```diff
@@ -0,0 +1,62 @@
+const domBookList = document.getElementById('book-list');
+const inputTitle = document.getElementById('input-title');
+const inputAuthor = document.getElementById('input-author');
+const buttonAdd = document.getElementById('button-add');
+
+const stringBookList = localStorage.getItem('bookList');
+
+let bookList = [];
+
+if (stringBookList) {
+  const parsedBookList = JSON.parse(stringBookList);
+  if (Array.isArray(parsedBookList)) {
+    bookList = parsedBookList;
+  }
+}
+
+function removeBook(index) {
+  bookList.splice(index, 1);
```

Comment:
# Optional
- Hi, I would advise you not to use [push](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push), [splice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice) methods because they mutate the array. Instead, you will need to get used to [concat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat), [filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter), [slice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice) method. They don't mutate the array.